src/graph/optimizer/OptimizerUtils.cpp

```diff
@@ -724,7 +208,15 @@ bool mergeRangeColumnHints(const ColumnDef& field,
       }
     }
   }
-  return !(*begin >= *end);
+  bool ret = true;
+  if (begin->first > end->first) {
+    ret = false;
+  } else if (begin->first == end->first) {
+    if (begin->second == false && end->second == false) {
```

Comment:
What will happen we pass interval of (x, x)? Report error?

Reply:
No errors will be reported and will be executed normally. But no data will be returned.

Reply:
👌